spec/unit/provider/package/zypper_spec.rb

```diff
@@ -136,7 +127,7 @@ def shell_out_expectation!(command, options=nil)
       shell_out_expectation!(
         "zypper --non-interactive install --auto-agree-with-licenses emacs=1.0"
```

Comment:
when i first read through this, I thought the `install` here was a bug (was expecting to read `update`, since we're in the upgrade section of the tests). 
Had to go through the code and actually test on the command-line that the `zypper install` actually does upgrade packages (even when a version isn't specified and there's a newer one available... : / 
Maybe we could add a line comment in the `upgrade_package` method of the zypper provider to explain why `install_package` is called, or am I the only one who would have thought `zypper install` wound not upgrade the package if it's already installed...?